src/api/app/views/webui/project/_tabs.html.erb

```diff
@@ -14,7 +14,7 @@
     <% else %> <!-- also for incident project -->
       <%= tab 'repositories', "Repositories", :controller => :project, :action => :repositories unless @project.is_remote? or @spider_bot %>
     <% end %>
-    <% if @project.api_obj.repositories && !@spider_bot -%>
+    <% if @project.repositories.any? && !@spider_bot -%>
```

Comment:
Not 100% sure if that can happen for a relation, assuming it is one, but if repositories would be nil this would cause an exception